contrib/ansible/dashboard/README.md

```diff
@@ -1,15 +1,23 @@
-## Ansible playbook for pbench dashboard
-This will ease installation, using pbench dashboard.
+## Ansible playbooks for installing and deploying the pbench dashboard
+This will ease installation, and deployment of the pbench dashboard.
 
 ## Required
 - Ansible needs to be installed on the host where you want to run this playbook
-- Python
-- JSON file containing the following key values: "elasticsearch", "production", "run_index", "prefix"
+- An inventory file containing the following key values defined:
+  - "`elasticsearch`", "`production`", "`run_index`", "`prefix`"
```

Comment:
We should also include the new key called `graphql` for referencing the Prisma environment deployed along with pbench dashboard.

Reply:
I don't believe we included this key in `config.js.j2` as part of PR #1137 but we should consider including it with this PR.

Reply:
Okay, I'll fix that, thanks

Reply:
Fixed.